pyaccel/utils.py

```diff
@@ -2,8 +2,6 @@
 
 import warnings
 import numpy as _numpy
-
-
```

Comment:
leave one empty between builtin modules/packages imports and external modules/packages imports  and one empty line between external modules/packages imports and our own modules imports:

```python3
import warnings

import numpy as _numpy
 
import trackcpp as _trackcpp
```

Reply:
Fixed, thanks!